js/web/script/pull-prebuilt-wasm-artifacts.ts

```diff
@@ -145,12 +145,14 @@ downloadJson(
                 extractFile(zip, WASM_FOLDER, 'ort-wasm-simd-threaded.wasm', folderName);
                 extractFile(zip, WASM_FOLDER, 'ort-wasm-simd.jsep.wasm', folderName);
                 extractFile(zip, WASM_FOLDER, 'ort-wasm-simd-threaded.jsep.wasm', folderName);
+                extractFile(zip, WASM_FOLDER, 'ort-training-wasm-simd.wasm', folderName);
```

Comment:
Can we make "pulling training artifacts" optional? In most cases where training is not required, users dont even need to pull these artifacts.

Reply:
alternatively, download if available. I am assuming, right now downloading will fail if the training artifact is not available.

Reply:
Similar to the JSEP & SIMD options, the training artifacts will be required for the full build. The training artifacts should be made available in the same PR because the training artifacts will be included in the pipeline build.